app/controllers/katello/remote_execution_controller.rb

```diff
@@ -3,6 +3,18 @@ module Katello
     class RemoteExecutionController < JobInvocationsController
       include Concerns::Api::V2::BulkHostsExtensions
 
+      def_param_group :bulk_errata_ids do
+        param :included, Hash, :desc => N_("Errata to exclusively include in the action"), :required => true, :action_aware => true do
+          param :search, String, :required => false, :desc => N_("Search string for erratum to perform an action on")
+          param :ids, Array, :required => false, :desc => N_("List of errata ids to perform an action on, (ex: RHSA-2019:1168)")
+        end
+        param :excluded, Hash, :desc => N_("Errata to explicitly exclude in the action."\
+                                           " All other applicable errata will be included in the action,"\
+                                           " unless an included parameter is passed as well."), :required => true, :action_aware => true do
+          param :ids, Array, :required => false, :desc => N_("List of errata ids to exclude and not run an action on, (ex: RHSA-2019:1168)")
+        end
+      end
```

Comment:
is it possible to refer to this from the other controller so it can be shared?

Reply:
I think you can reference this from another controller like this:

param_group :search, Api::V2::ApiController

Or you could put it into the HostErrataExtensions module to share it (i think i like that route better)